src/DataBoxEdge/DataBoxEdge.Test/ScenarioTests/Common.ps1

```diff
@@ -48,7 +48,8 @@ Returns standard EncryptionKey
 #>
 function Get-EncryptionKey
 {
-	$encryptionKey = ConvertTo-SecureString -String ""
+	$val = Get-AzKeyVaultSecret -VaultName azpsdbe -Name "EncryptionKey"
+	$encryptionKey = $val.SecretValueText
```

Comment:
why not return "$val.SecretValueText" directly

Reply:
Sure, will do